centreon/src/Core/Host/Infrastructure/Repository/DbWriteHostRepository.php

```diff
@@ -0,0 +1,496 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Host\Infrastructure\Repository;
+
+use Centreon\Domain\Log\LoggerTrait;
+use Centreon\Infrastructure\DatabaseConnection;
+use Core\Common\Application\Converter\YesNoDefaultConverter;
+use Core\Common\Domain\HostType;
+use Core\Common\Infrastructure\Repository\AbstractRepositoryRDB;
+use Core\Common\Infrastructure\Repository\RepositoryTrait;
+use Core\Common\Infrastructure\RequestParameters\Normalizer\BoolToEnumNormalizer;
+use Core\Host\Application\Converter\HostEventConverter;
+use Core\Host\Application\Repository\WriteHostRepositoryInterface;
+use Core\Host\Domain\Model\NewHost;
+
+class DbWriteHostRepository extends AbstractRepositoryRDB implements WriteHostRepositoryInterface
+{
+    use LoggerTrait, RepositoryTrait;
```

Comment:
PSR-12 : one by line.
```suggestion
    use LoggerTrait;
    use RepositoryTrait;
```